test/ex_doc/formatter/html/templates_test.exs

```diff
@@ -148,14 +148,21 @@ defmodule ExDoc.Formatter.HTML.TemplatesTest do
       ## Example:
       """
 
-      assert Templates.synopsis(Earmark.as_html!(doc1)) ==
+      assert Templates.synopsis(to_html(doc1)) ==
```

Comment:
Something else changed in Earmark that caused this regression in our test suite around rendering paragraphs: https://github.com/elixir-lang/ex_doc/pull/1183#issuecomment-640247967

With this change we make the test better, we don't use Earmark to generate html anymore but our own custom code, so this way it's closer to reality too.